go/api/command/command.proto

```diff
@@ -126,7 +126,31 @@ message InputSpec {
   SymlinkBehaviorType.Value symlink_behavior = 6;
 
   // Node properties of inputs.
-  map<string,build.bazel.remote.execution.v2.NodeProperties> input_node_properties = 7;
+  map<string,NodeProperties> input_node_properties = 7;
+}
+
+// A copy of NodeProperties from https://github.com/bazelbuild/remote-apis/blob/main/build/bazel/remote/execution/v2/remote_execution.proto
+// to importing it as a proto dependency
```

Comment:
Is it a worth mentioning the issue on the high spec machine? 
Is it ok to put http://b/296368945 here?
At least, this build failure is public. https://luci-milo.appspot.com/ui/p/chromium/builders/ci/linux-build-perf-developer/3291/overview

Reply:
This comment has been addressed.